app/scripts/session/sessionServices.js

```diff
@@ -76,6 +79,7 @@ window.vcSession = window.vcSession ? window.vcSession : {};
           }
         });
       },
+      // TODO: should we send request without validating userID?
       loadUserVirtualCohorts: function(userID) {
```

Comment:
Actually this function is specific to cbio. So I dont want to pass this attribute to service. Thinking to get user if from spring session and make request to session-service api.

BTW this function would be only used for secured cbioportal instance(not for public instance)